tools/python/serializer.py

```diff
@@ -0,0 +1,28 @@
+# License: BSD 3 clause
```

Comment:
I would rather put this kind of utility closer to the fetch_tick_dataset method. I also think that it should be a method rather than a script.

Reply:
sure